torch/distributed/tensor/parallel/style.py

```diff
@@ -702,6 +666,9 @@ def __init__(
         output_layouts: LayoutsType = Shard(0),
         use_local_output: bool = True,
     ) -> None:
+        if not isinstance(output_layouts, tuple):
```

Comment:
why only do this for `output_layouts`? I think we should do this to `input_layouts` too, please add a test on this

Reply:
`input_layouts` is not used in this case.